public/he.h

```diff
@@ -982,14 +996,12 @@ he_padding_type_t he_ssl_ctx_get_padding_type(he_ssl_ctx_t *ctx);
  */
 he_return_code_t he_ssl_ctx_set_aggressive_mode(he_ssl_ctx_t *ctx);
 
-#ifndef HE_NO_PQC
```

Comment:
This got removed was because a `python/make_header.py` bug. I'll fix it in next commit. cc @expressvpn-raihaan-m

Reply:
It's fixed now.

Reply:
Can we squash the second commit into the first commit please? There's no need to commit this error into the git history forever.

Reply:
Instead of squashing, I think better way is to cherry pick the 'make_header.py' fix into a separate PR and merge that first. I'll do it tomorrow.